controlllers/errorHandlingController.js

```diff
@@ -0,0 +1,11 @@
+exports.handle404Statuses = (error, req, res, next) => {
```

Comment:
We don't handle 404's with error first middleware - this should be for other error status'. You've already handled generic 404's "path not found". You can also refactor this to be a generic error handling function.